application/db/pg/start.js

```diff
@@ -4,4 +4,12 @@ async () => {
   }
   const options = { ...config.database, console };
   db.pg = new metarhia.metasql.Database(options);
+
+  // Check that we connected successfully.
+  await db.pg.query('SELECT now()').catch((err) => {
```

Comment:
Can we somehow detect connection status without a query?

Reply:
To example, we can catch connection errors by manually creating connection using connect method from Pool:
```JS
await db.pg.pool.connect().catch((err) => {
....
});